### PR TITLE
3 agregar home screen

### LINKED ISSUE #3: Agregar screen del home

### PR DESCRIPTION
**Pull request by MauriNestor**

### CODE REVIEW

File: app/src/main/java/org/scesi/cappuchino/ui/screens/homeScreen.kt [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.scesi.cappuchino.R
+
+@Composable
+fun HomeScreen() {
+    Column(
+        modifier = Modifier
+//            .background(Color.Gray)
```
Comment (beldier) [inline app/src/main/java/org/scesi/cappuchino/ui/screens/homeScreen.kt:41]: Do not commit comments without explanation, leave an explanation or delete it

File: app/src/main/java/org/scesi/cappuchino/ui/screens/homeScreen.kt [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+        verticalArrangement = Arrangement.Top
+
+    ) {
+        Text(
+            text = "Ponte comodo y arma tu horario con un cappuchino",
+            textAlign = TextAlign.Center,
+            fontSize = 15.sp,
+            modifier = Modifier
+                .wrapContentHeight()
+//                .background(Color.Cyan)
```
Comment (beldier) [inline app/src/main/java/org/scesi/cappuchino/ui/screens/homeScreen.kt:54]: Do not commit comments without explanation, leave an explanation or delete it

File: app/src/main/java/org/scesi/cappuchino/ui/screens/homeScreen.kt [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+        modifier = Modifier
+//            .background(Color.Gray)
+            .fillMaxSize()
+            .padding(20.dp),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Top
+
+    ) {
+        Text(
+            text = "Ponte comodo y arma tu horario con un cappuchino",
```
Comment (beldier) [inline app/src/main/java/org/scesi/cappuchino/ui/screens/homeScreen.kt:49]: Dont use raw strings, instead use string resources , extract it (option + enter)

File: app/src/main/java/org/scesi/cappuchino/ui/screens/homeScreen.kt [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+                .wrapContentHeight()
+//                .background(Color.Cyan)
+                .padding(top = 20.dp)
+
+        )
+        Spacer(modifier = Modifier.height(16.dp))
+
+        Image(
+            painter = painterResource(id = R.drawable.cap),
+            contentDescription = "descrip",
```
Comment (beldier) [inline app/src/main/java/org/scesi/cappuchino/ui/screens/homeScreen.kt:62]: Dont use raw strings, instead use string resources , extract it (option + enter)

File: app/src/main/java/org/scesi/cappuchino/ui/screens/homeScreen.kt [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun SearchBar(){
+    var searchText by remember { mutableStateOf("") }
+
+    OutlinedTextField(
+        value = searchText,
+        onValueChange = { searchText = it },
+        placeholder = { Text("Busca tu carrera", fontSize = 10.sp) },
```
Comment (beldier) [inline app/src/main/java/org/scesi/cappuchino/ui/screens/homeScreen.kt:119]: Dont use raw strings, instead use string resources , extract it (option + enter)

File: app/src/main/java/org/scesi/cappuchino/ui/screens/homeScreen.kt [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+        trailingIcon = {
+            Icon(
+                painter = painterResource(id = R.drawable.lupaicon),
+                contentDescription = "Buscar",
```
Comment (beldier) [inline app/src/main/java/org/scesi/cappuchino/ui/screens/homeScreen.kt:123]: Dont use raw strings, instead use string resources , extract it (option + enter)

File: app/src/main/java/org/scesi/cappuchino/ui/screens/homeScreen.kt [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+                modifier = Modifier.size(24.dp),
+                tint = Color.Gray
+            )
+        },
+        modifier = Modifier
+            .fillMaxWidth()
+            .wrapContentHeight()
+            .padding(horizontal = 5.dp)
+//            .background(Color.Yellow)
```
Comment (beldier) [inline app/src/main/java/org/scesi/cappuchino/ui/screens/homeScreen.kt:132]: Same with comments

File: app/src/main/java/org/scesi/cappuchino/ui/Cappuchino.kt [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+    CappuchinoScreen {
+        CappuchinoNavigation(
+            navHostController = appState.navController
+        )
+    }
+}
+
+
+@Composable
+fun CappuchinoScreen(content: @Composable ()->Unit){
```
Comment (beldier) [inline app/src/main/java/org/scesi/cappuchino/ui/Cappuchino.kt:20]: format code  (option + command + L )

**Comment by beldier** [comment]
@MauriNestor  please take a look at the comments

**Comment by MauriNestor** [comment]
I think all the comments has been resolved.

File: app/src/main/java/org/scesi/cappuchino/ui/Cappuchino.kt (hunk 8 above)
Comment (MauriNestor) [reply app/src/main/java/org/scesi/cappuchino/ui/Cappuchino.kt]: done @beldier

**Review by rybeler — CHANGES_REQUESTED** [review]
- Fix font size 
 - Make the list scrollable 
 - Correct the page message

**Comment by MauriNestor** [comment]
> * Fix font size
> * Make the list scrollable
> * Correct the page message 

all the comments corrected @beldier

**Comment by MauriNestor** [comment]
@beldier I did preview, content and homeScreen funtion and also used dimens instead of magic numbers

**Review by rybeler — APPROVED** [review]

**Pull request merged**